zip-0316.rst

```diff
@@ -664,6 +710,11 @@ Change level is non-hardened.
 Deriving a UIVK from a UFVK
 ---------------------------
 
+As of `Revision 1`_, a Consumer of a UIVK MUST understand the
+meaning of all "MUST-understand" Metadata Item typecodes present in the
+UFVK in order to perform UIVK derivation, and consumers MUST follow the
+derivation directives specified for each such Metadata Item.
```

Comment:
```suggestion
UFVK in order to perform UIVK derivation. For Metadata Items recognised
by the Consumer, the processing of the Item when deriving a UIVK is
specified in the section or ZIP describing that Item.
```
The difference here is that we're saying that such processing might be specified (and where to look for the specification) for all Metadata Items, not just MUST-understand ones. It's not necessary to use MUST here: the relevant section or ZIP will say what the conformance requirement is, and it might not be a MUST in general (but it is for 0xE0 and 0xE1).